Resources/Prototypes/_RMC14/Entities/Clothing/OuterClothing/cmb.yml

```
@@ -0,0 +1,69 @@
+- type: entity
+  parent: RMCBaseMarineArmorLight
+  id: CMArmorM4RLight
```

```suggestion
  id: RMCArmorM4RLight
```
Use `RMC` instead of `CM` for new prototypes.